string_assignment.js

```diff
@@ -0,0 +1,164 @@
+// # String Exercises
```

Comment:
I like that fact that you experimented with different strong methods. Good demonstration of string methods and properties to solve problems. Keep up the good work!